org-code-javabuilder/playground/src/main/java/org/code/playground/Item.java

```diff
@@ -1,53 +1,67 @@
 package org.code.playground;
 
 public abstract class Item {
-  Item(int x, int y, int height) {}
+  private int xLocation;
+  private int yLocation;
+  private int height;
+
+  Item(int x, int y, int height) {
```

Comment:
should item also take a width?

Reply:
or is there no width because `TextItem` doesn't have a width?

Reply:
Yeah, no width in `TextItem`, at least in spec. That could be wrong though?

Reply:
I think that makes sense, since the width is determined by the text itself

Reply:
> I think that makes sense, since the width is determined by the text itself

Yeah, I believe this is intended